components/demo/code-view.js

```diff
@@ -27,7 +27,7 @@ class CodeView extends LitElement {
 	render() {
 		return html`
 			<div class="d2l-code-view-src"><slot @slotchange="${this._handleSlotChange}"></slot></div>
-			<div language="${this.language}" class="d2l-code-view-code">${this._codeTemplate}</div>
+			<div class="d2l-code-view-code">${this._codeTemplate}</div>
```

Comment:
This wasn't doing anything.

Reply:
Currently I think this is used by:
```
:host .d2l-code-view-code::before {
    content: attr(language);
}
```

The language shows here:
![image](https://user-images.githubusercontent.com/9042472/82825630-94a08280-9e79-11ea-9f27-6d99e136c3f6.png)

Maybe we can just change it to a data attribute?

Reply:
Ah cool, I was having trouble figuring out how it got applied. Will fix!

Reply:
K fixed!